src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -32,20 +32,31 @@
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output metrics file.")
     public File OUTPUT;
 
-    @Option(shortName = "MQ", doc = "Minimum mapping quality for a read to contribute coverage.", overridable = true)
-    public int MINIMUM_MAPPING_QUALITY = 20;
+    @Option(shortName = "USABLE_MQ", doc = "Minimum mapping quality for a read to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_MAPPING_QUALITY = 20;
 
-    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
-    public int MINIMUM_BASE_QUALITY = 20;
+    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_BASE_QUALITY = 20;
+
+    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_MAPPING_QUALITY = 0;
+
+    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_BASE_QUALITY = 3;
```

Comment:
all except Q2...this is a metric that Illumina uses and some people are
using their metric for comparison. don't get me started about it...

On Tue, Dec 22, 2015 at 1:45 PM, Nils Homer <notifications@github.com>
wrote:

> In src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java
> <https://github.com/broadinstitute/picard/pull/365#discussion_r48284030>:
>
> >
> > -    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
> > -    public int MINIMUM_BASE_QUALITY = 20;
> > +    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true)
> > +    public int MINIMUM_USABLE_BASE_QUALITY = 20;
> > +
> > +    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true)
> > +    public int MINIMUM_RAW_MAPPING_QUALITY = 0;
> > +
> > +    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true)
> > +    public int MINIMUM_RAW_BASE_QUALITY = 3;
>
> why 3 for raw (which I interpret to mean all)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/picard/pull/365/files#r48284030>.
>